### PR TITLE
add rub cat belly option to cuddle cat story

### PR DESCRIPTION
**Pull request by sunnymui**

### CODE REVIEW

**Comment by sunnymui** [comment]
Please review this, @casey-collab

File: english/cuddle-the-cat/cuddle-cat.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 You pick up your kitty and cuddle her for hours. She happily purrs. And
 you happily purr back. Your cat looks at you like you're crazy, but what
 are you going to do?
+
+[Try to rub your cat's belly](/rub-cat-belly/rub-cat-belly.md)
```
Comment (casey-collab) [inline english/cuddle-the-cat/cuddle-cat.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/cuddle-the-cat/rub-cat-belly/rub-cat-belly.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Your cat flops down on the floor with his belly sticking out.
```
Comment (casey-collab) [inline english/cuddle-the-cat/rub-cat-belly/rub-cat-belly.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/cuddle-the-cat/rub-cat-belly/rub-cat-belly.md (hunk 2 above)
Comment (sunnymui) [inline english/cuddle-the-cat/rub-cat-belly/rub-cat-belly.md]: Please review this, @casey-collab

**Comment by sunnymui** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**